cross/armel/toolchain.cmake

```diff
@@ -16,7 +16,7 @@ add_compile_options(--sysroot=${CROSS_ROOTFS})
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -target ${TOOLCHAIN}")
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} --sysroot=${CROSS_ROOTFS}")
 
-if("${CLI_CMAKE_RUNTIME_ID}" MATCHES "tizen.*")
```

Comment:
This is a CMake define - can you please help explain why it does not get propagated to the custom toolchain you point it to for Tizen build?